sda/cmd/s3inbox/proxy.go

```diff
@@ -273,7 +304,7 @@ func (p *Proxy) prependBucketToHostPath(r *http.Request) {
 	// Extract username for request's url path
 	str, err := url.ParseRequestURI(r.URL.Path)
 	if err != nil || str.Path == "" {
-		log.Errorf("failed to get path from query (%v)", r.URL.Path)
+		log.Warningf("failed to get path from query (%v)", r.URL.Path)
```

Comment:
Here we need to return an error because if `str.Path` is `""` the `strings.Split` below will fail with a `nil pointer dereference` error.

Reply:
good catch! implemented in 7d8f023